util/crs_rules_check/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Can we remove this file? The global repo already has the apache license, I don't think we need it here.

Reply:
We should talk about this on next meeting (sub-module or not). I think so we can live that together.

Reply:
I removed the LICENSE file.